flink-libraries/flink-cep-scala/pom.xml

```diff
@@ -93,6 +93,8 @@ under the License.
     </dependencies>
 
     <build>
+        <!-- Not strictly necessary because of the build-helper-maven-plugin below. -->
+        <sourceDirectory>src/main/scala</sourceDirectory>
```

Comment:
Yes, one of the two works. The easiest fix is this one liner though I'm not sure about Eclipse.